<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KUserFeedback::Console::SchemaEditor</class>
 <widget class="QWidget" name="KUserFeedback::Console::SchemaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="KUserFeedback::Console::SchemaEditWidget" name="schema">
      <attribute name="title">
       <string>Schema</string>
      </attribute>
     </widget>
     <widget class="KUserFeedback::Console::AggregationEditWidget" name="aggregation">
      <attribute name="title">
       <string>Aggregation</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>&amp;Save Schema</string>
   </property>
   <property name="toolTip">
    <string>Save changes to the product schema.</string>
   </property>
  </action>
  <action name="actionImportSchema">
   <property name="icon">
    <iconset theme="document-import"/>
   </property>
   <property name="text">
    <string>&amp;Import Schema...</string>
   </property>
   <property name="toolTip">
    <string>Import a schema for this product.</string>
   </property>
  </action>
  <action name="actionExportSchema">
   <property name="icon">
    <iconset theme="document-export"/>
   </property>
   <property name="text">
    <string>&amp;Export Schema...</string>
   </property>
   <property name="toolTip">
    <string>Export the schema of this product.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUserFeedback::Console::SchemaEditWidget</class>
   <extends>QWidget</extends>
   <header>schemaeditor/schemaeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUserFeedback::Console::AggregationEditWidget</class>
   <extends>QWidget</extends>
   <header>schemaeditor/aggregationeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
